src/lib/common/traits.hpp

```diff
@@ -1077,6 +1077,12 @@ inline T const& escape(T const& x) {
 }
 //! @}
 
+/**
+ * @brief Declares that the type @param T is usable as an output stream. 
+ * Enabled by default for all subtypes of std::ostream.
+ */
+template <typename T>
+struct is_ostream : public std::is_base_of<std::ostream, T> {};
```

Comment:
I see it's used in field.hpp, but don't get what's the purpose w.r.t. custom ostreams

Reply:
> I see it's used in field.hpp, but don't get what's the purpose w.r.t. custom ostreams

It is necessary to avoid overload collision with serialising operators, which are also based on `<<` but with stream-like classes for which `is_ostream` is false